public/components/model_version/model_version.tsx

```diff
@@ -29,14 +30,20 @@ import { ModelVersionInformation } from './version_information';
 import { ModelVersionArtifact } from './version_artifact';
 import { ModelVersionTags } from './version_tags';
 import { ModelVersionFormData } from './types';
+import { ModelGroupSearchItem } from '../../apis/model_group';
```

Comment:
Is this PR based the latest `feature/model-registry` branch? I think the `apis/model_group` won't exists in that branch.